app/src/main/res/values/strings.xml

```diff
@@ -522,6 +522,8 @@
     <string name="collection_open_tabs">Open tabs</string>
     <!-- Text for the menu button to remove a top site -->
     <string name="remove_top_site">Remove</string>
+    <!-- Text for the menu button to delete a top site from history -->
+    <string name="delete_from_history">Delete from History</string>
```

Comment:
Should `History` be capitalized here?

Reply:
Checked with Vesta and renamed it to "Delete from history". Thank you for the nudge!